mappings/net/minecraft/network/packet/c2s/play/BookUpdateC2SPacket.mapping

```diff
@@ -1,5 +1,13 @@
 CLASS net/minecraft/class_2820 net/minecraft/network/packet/c2s/play/BookUpdateC2SPacket
 	FIELD field_26897 slot I
+	FIELD field_34041 bookPages Ljava/util/List;
+	FIELD field_34042 bookTitle Ljava/util/Optional;
```

Comment:
Maybe just `pages` `titles` as this packet is already about book (so it's redundant info)?